.github/workflows/ci_flutter_example.yml

```
@@ -36,19 +23,36 @@ jobs:
         with:
           rust: true
           flutter: true
```

🔧 here we still need the token because we use the old setup script
```suggestion
          flutter: true
          github_token: ${{ secrets.GITHUB_TOKEN }}
```